### ISSUE TITLE
Works great on wsl Ubuntu 20.04 but fails on Intel NUC also on Ubuntu 20.04

### ISSUE THREAD

**Issue by adrian-basheer** [Ricciolo/libfacedetection.net#1]
It fails on calling .Detect when running on an NUC machine running Ubuntu 20.04. 

Works great when publishing to both windows and WSL Ubuntu 20.04.

Has anyone had this issue?

Best regards,
Adrian.

**Comment by Ricciolo** [comment]
Could you give me more details of the error?

**Comment by adrian-basheer** [comment]
I have the generated dump file but I cannot really understand where the problem is.  I am sure it is on the detect line because the second console write does not appear:

I am using dotnet core 5.

                Console.WriteLine("Detecting faces");
                IReadOnlyList<CnnFaceDetected> result = faceDetector.Detect(im, new System.Drawing.Size(1280, 720));
                Console.WriteLine("Detected faces");

would providing the top part of the dump file help?

Thank you so much for your quick response. Much appreciated.

Sorry, the error i see on the console is "Illegal instruction (core dumped)"

**Comment by adrian-basheer** [comment]
As an update to the above, this may be due to the test machine having a Celeron processor, which may not support the instructions needed.

**Comment by Ricciolo** [comment]
Yep, I confirm that the library is compiled with AVX2 enabled, in order to speed up the process, and older celeron does not support it
